common/cli-utils.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+require("ts-node/register"); /* enable node during migration */
 require('dotenv').config();
```

Comment:
let all scripts import `common/cli-utils.js` so that this backwards compat doesn't need to be added and maintained everywhere? We already make use of this for `require('dotenv').config();`